interface/practice/ins_search.php

```diff
@@ -168,13 +168,13 @@ function validate(f) {
    $ins_id = generate_id();
 
    sqlInsert("INSERT INTO insurance_companies ( " .
-    "id, name, attn, cms_id, freeb_type, x12_receiver_id, x12_default_partner_id " .
+    "id, name, attn, cms_id, ins_co_type, x12_receiver_id, x12_default_partner_id " .
```

Comment:
here are other places that will also error out:
``````
[01:16][brady2@nonsense:~/git/openemr((HEAD detached at wakie87/freeb))]$ grep -Ri "ins_co_type" 
/var/www/openemr/interface/practice/ins_search.php: $ins_co_type_array = array(''
/var/www/openemr/interface/practice/ins_search.php:    "id, name, attn, cms_id, ins_co_type, x12_receiver_id, x12_default_partner_id " .
/var/www/openemr/interface/practice/ins_search.php:    "'" . add_escape_custom($_POST['form_ins_co_type']) . "', " .
/var/www/openemr/interface/practice/ins_search.php:   <select name='form_ins_co_type'>
/var/www/openemr/interface/practice/ins_search.php: for ($i = 1; $i < count($ins_co_type_array); ++$i) {
/var/www/openemr/interface/practice/ins_search.php:  // if ($i == $row['ins_co_type']) echo " selected";
/var/www/openemr/interface/practice/ins_search.php:  echo ">" . $ins_co_type_array[$i] . "\n";
/var/www/openemr/custom/qrda_functions.php:			$insQry = "SELECT insd.*, ic.ins_co_type FROM (SELECT pid, provider FROM insurance_data WHERE type = 'primary' ORDER BY id DESC) insd ".
/var/www/openemr/custom/qrda_functions.php:				if($insRow['ins_co_type'] == 8){//Self Pay (Private Insurance)
/var/www/openemr/custom/qrda_functions.php:				}else if($insRow['ins_co_type'] == 2){//Medicare
/var/www/openemr/custom/qrda_functions.php:				}else if($insRow['ins_co_type'] == 3){//Self Pay (Private Insurance)
/var/www/openemr/custom/qrda_functions.php:		$insQry = "SELECT insd.*, ic.ins_co_type FROM (SELECT pid, provider FROM insurance_data WHERE type = 'primary' ORDER BY id DESC) insd ".
/var/www/openemr/custom/qrda_functions.php:			if($insRow['ins_co_type'] == 8){//Self Pay (Private Insurance)
/var/www/openemr/custom/qrda_functions.php:			}else if($insRow['ins_co_type'] == 2){//Medicare
/var/www/openemr/custom/qrda_functions.php:			}else if($insRow['ins_co_type'] == 3){//Self Pay (Private Insurance)
/var/www/openemr/custom/export_xml.php:  "c1.name AS name1, c1.ins_co_type AS instype1, " .
/var/www/openemr/custom/export_xml.php:  "c2.name AS name2, c2.ins_co_type AS instype2, " .
/var/www/openemr/custom/export_xml.php:  "c3.name AS name3, c3.ins_co_type AS instype3, " .
``````